apisix/plugins/elasticsearch-logger.lua

```diff
@@ -31,10 +32,19 @@ local batch_processor_manager = bp_manager_mod.new(plugin_name)
 local schema = {
     type = "object",
     properties = {
+        -- deprecated, use "endpoint_addrs" instead
         endpoint_addr = {
             type = "string",
             pattern = "[^/]$",
         },
+        endpoint_addrs = {
```

Comment:
Extending endpoint_addr in a compatible way, and adding a new configuration is not a good choice

Reply:
Thank you for your reply.

What is your suggestion?
Can you give an example?
I refer to PR #7517

> feat(clickhouse-logger): support multi clickhouse endpoints

Reply:
Looking at this, it also indicates the need to unify into one. You can refer to the definition of nodes_schema to allow endpoint_addrs to be set to multiple types: https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L289

Reply:
> Looking at this, it also indicates the need to unify into one. You can refer to the definition of nodes_schema to allow endpoint_addrs to be set to multiple types: https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L289

Thanks for the helpful advice!

Reply:
Hi @soulbird. 
I fixed the code according to your suggestion. please review it.